recipients_app/README.md

```diff
@@ -31,7 +32,15 @@ setup.
 
 ## Rebuilding JSON Serialization
 
-flutter pub run build_runner watch --delete-conflicting-outputs
```

Comment:
The terminal told me that "flutter pub run" is deprecated and you should use "dart run" instead.